5eShaped/precompiled/translations/de.json

```diff
@@ -55,17 +55,17 @@
     "athletics": "Athletik",
     "deception": "Täuschung",
     "history": "Geschichte",
-    "insight": "Einsicht",
-    "intimidation": "Einschüchterung",
-    "investigation": "Investigation",
+    "insight": "Motiv erkennen",
+    "intimidation": "Einschüchtern",
```

Comment:
Neverwinter Nights (based on 3.5) used "Entdecken", thats why I chose it too. Both seem fine I guess.